sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts

```diff
@@ -426,7 +427,29 @@ describe(
           },
         },
       });
-      expect(notification).toBe(null);
+      expect(createdNotification.messagePayload).toStrictEqual({
+        template_id: createdNotification.notificationMessage.templateId,
+        email_address: TEST_EMAIL,
+        personalisation: {
+          application_file: {
+            file:
+              "RnVsbCBUaW1lIEJDIFN0dWRlbnQgTG9hbixGdWxsIFRpbWUgQkMgU3R1ZGVudCBHcmFudCxGdWxsIFRpbWUgQkMgVG90YWwsUGFydCBUaW1lIEJDIFN0dWRlbnQgR3JhbnQsUGFydCBUaW1lIEJDIFRvd" +
+              "GFsLEJDIFRvdGFsLFRvdGFsIFJlY29yZHMsRmlsZSBEYXRlLEJhdGNoIFJ1biBEYXRlLFNlcXVlbmNlIE51bWJlcg0KMCwwLDAsMCwwLDAsMCwyMDI0LTAxLTMxLCwzMjI4",
+            filename: `Daily_Disbursement_File_${FILE_DATE}_${SEQUENCE_NUMBER}.csv`,
+            sending_method: "attach",
+          },
+        },
+      });
+      // Verify the file content as expected.
+      const file =
+        createdNotification.messagePayload["personalisation"][
+          "application_file"
+        ]["file"];
+      const fileContent = Buffer.from(file, "base64").toString("ascii");
+      expect(fileContent).toContain(
+        "Full Time BC Student Loan,Full Time BC Student Grant,Full Time BC Total,Part Time BC Student Grant,Part Time BC Total,BC Total,Total Records,File Date,Batch Run Date,Sequence Number",
+      );
+      expect(fileContent).toContain("0,0,0,0,0,0,0,2024-01-31,,3228");
```

Comment:
We can build the expected bas64 from the string `0,0,0,0,0,0,0,2024-01-31,,3228` and assert created notification.